python/wpt/update.py

```diff
@@ -126,6 +126,8 @@ def run_update(**kwargs) -> bool:
 
 def create_parser(**_kwargs):
     parser = wptcommandline.create_parser_update()
+    # TODO: These two default values should be `True` and `False` to align with
+    #       `./mach test-wpt` command.
```

Comment:
I really think we should change all scripts at once and not do this in a piecemeal way. It's very confusing when different scripts have different behaviors.

Reply:
Hm, that makes sense.

Are you referring to all scripts that have both the "--layout-2013" and "--layout-2020" options?
Since `./mach` appears to have many commands, I'm not sure I can handle all of them. (Or are there not so many commands to change?) But I'd be glad to take a look.

Reply:
Yep, I'm referring to all commands that have those two options. A way to find them all is to grep the `python` directory for these strings. I think there are maybe 3 or 4 in total.

Reply:
It seems that only `test-wpt(-xxx)` and `update-wpt` commands are related here, if I understand correctly.